san/sanbase_graphql.py

```diff
@@ -611,6 +577,15 @@ def _transform_query_args(**kwargs):
     kwargs['from_date'] = _format_from_date(kwargs['from_date'])
     kwargs['to_date'] = _format_to_date(kwargs['to_date'])
 
+    return_fields = []
+    if 'return_fields' in kwargs:
+        _format_return_fields(return_fields, kwargs['return_fields'])
+        kwargs['return_fields'] = return_fields
```

Comment:
We can we simplify like this: `kwargs['return_fields'] = format_return_fields(kwargs['return_fields'])`